app/controllers/actions/DependentTasksAction.scala

```diff
@@ -30,7 +30,13 @@ class DependentTasksActionImpl @Inject() (implicit val executionContext: Executi
     with Logging {
 
   override protected def filter[A](request: DataRequest[A]): Future[Option[Result]] = {
-    val incompleteTasks = config.dependentTasks.filterNot(request.userAnswers.tasks.get(_).exists(_.isCompleted))
+    val incompleteTasks = config.dependentTasks.filterNot(
+      request.userAnswers.tasks
+        .get(_)
+        .exists(
+          tasks => tasks.isCompleted || tasks.isError
```

Comment:
Is this correct? I'd have thought a dependent task is completed if is either `Completed` or `Unavailable`? If that's the case I'd update the `isCompleted` method in `TaskStatus` rather than here

Reply:
This was an issue I noticed if every journey was in error state and you tried to edit transport details or items first it wouldn't let you in, as it thought it was not completed. But my line of thinking is all the data is there in error state and it is technically a complete section - if that makes sense?

I will update the .isCompleted as it seems it is only used for this section of code so shouldn't have any negative effects